packages/plugin-client-drizzle/README.md

```diff
@@ -0,0 +1,59 @@
+# @xata.io/drizzle
+
+A [Drizzle](https://github.com/drizzle-team/drizzle-orm) driver for [Xata](https://xata.io), using the [Xata serverless driver for TypeScript](https://github.com/xataio/client-ts).
+
+The goal of this package is polish the implementation and contribute it back to the official Drizzle repository.
+
+## Installation
+
+You should install both `drizzle-orm` and `@xata.io/drizzle` with `@xata.io/client`. You can install them with your favorite package manager:
+
+```bash
+# with pnpm
+pnpm add drizzle-orm @xata.io/drizzle @xata.io/client
+
+# with yarn
+yarn add drizzle-orm @xata.io/drizzle @xata.io/client
+
+# with npm
+npm install drizzle-orm @xata.io/drizzle @xata.io/client
+```
+
+## Usage
+
+To work with drizzle you need to define your models and then create a drizzle instance with the models.
+
+```ts
+import { pgTable, text } from 'drizzle-orm/pg-core';
+import { drizzle } from '@xata.io/drizzle';
+import { getXataClient } from './xata';
+
+const xata = getXataClient();
+const db = drizzle(xata);
+
+const drivers = pgTable('drivers', {
+  id: text('id').primaryKey(),
+  surname: text('surname'),
+  forename: text('forename')
+});
+
+const result = await db.select().from(drivers).execute();
```

Comment:
nit: Could we show a where clause here?

Reply:
`  .where(eq('nationality', 'spanish'))` I think should work, with `eq` imported from `drizzle-orm`.

Reply:
Updated with a more complex example